llvm/test/Transforms/LoopVectorize/X86/vplan-based-cost-assertion.ll

```diff
@@ -0,0 +1,87 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt < %s -passes=loop-vectorize -mtriple=x86_64-unknown-linux-gnu -S | FileCheck %s
+
+; Check if the vplan-based cost model select same VF to the legacy cost model.
```

Comment:
Might be good to group with other tests for various crashes, llvm/test/Transforms/LoopVectorize/X86/cost-model.